app/serializers/recipe_show_serializer.rb

```diff
@@ -1,8 +1,8 @@
 class RecipeShowSerializer < RecipeSerializer
   NIL_RESULT = 'unknown'.freeze
 
-  attributes :cooking_time, :video_link, :level
-  
+  attributes :level, :video_link, :cooking_time
+  has_many :rates
```

Comment:
I think that you should return `avg_rate`.  Returning collection of votes is pointless. You can fix that in a fast way.

Reply:
Good point Marcin 👌 having avg_rate and maybe rates_count makes more sense